OverTool/List/ListGameMode.cs

```diff
@@ -83,7 +83,16 @@ public class ListGameMode : IOvertool {
                     string name = Util.GetString(gm.Header.name, map, handler);
                     if (name != null) {
                         string desc = Util.GetString(gm.Header.description, map, handler);
-                        Console.Out.WriteLine(name);
+                        if (map.ContainsKey(gm.Header.difficultyName.key)) {
```

Comment:
`Util.GetString` handles map checking. Code should look like:

```csharp
string desc = Util.GetString(gm.Header.description, map, handler);
string difficultyName = Util.GetString(gm.Header.difficultyName, map, handler);
string difficultyDescription = Util.GetString(gm.Header. difficultyDescription, map, handler);
```

then follow the same code that is used to output `desc`

**N.B.** OWRecord doubles as a long, no need to use `.key`, the only reason some code uses it is because I only implemented that late into development